inference-engine/src/gna_plugin/gna_plugin.cpp

```diff
@@ -657,6 +658,7 @@ void GNAPlugin::LoadNetwork(CNNNetwork & _network) {
         CNNNetwork clonedNetwork = InferenceEngine::cloneNetwork(_network);
         const auto& graph = clonedNetwork.getFunction();
         ngraph::pass::Manager manager;
+        manager.register_pass<ngraph::pass::SwapInputMatMul>();
```

Comment:
I suppose this transformation is GNA specific so its code should be moved to GNA plugin